content/blog/announcing-keynote-speaker-for-pycon-india-2017-ola.md

```diff
@@ -0,0 +1,29 @@
+Title: Announcing Keynote Speakers for PyCon India 2017 - Ola Sitarska
+Date: 2017-08-28 18:23
+Category: 2017
+Slug: announcing-keynote-speakers-for-pycon-india-2017
+Summary: Announcing PyCon India 2017 Keynote Speakers
+
+We are proud to announce **Ola Sitarska** as one of the Keynote Speakers for PyCon India 2017.
+
+###Ola Sitarska
+
+![Ola Sitarka](https://in.pycon.org/2017/images/speakers/ola-sitarski.jpg)
```

Comment:
This picture is too large. Here's a fix.
- Resize it to 500 by 500 px
- Upload the image to /themes/inpycon2017/static/images folder
- Link the image using https://in.pycon.org/blog/theme/images/